spec/lib/root_route_constraint_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe RootRouteConstraint do
+  describe '#matches?' do
+
```

Comment:
Please move ``constraint = RootRouteConstraint.new`` to a ``let`` as you use it in every single test